src/librustc/mir/interpret/value.rs

```diff
@@ -415,48 +415,60 @@ impl<'tcx, Tag> Scalar<Tag> {
         }
     }
 
+    fn to_unsigned_with_bit_width(self, bits: u64) -> InterpResult<'static, u128> {
```

Comment:
Please mark this as `#[inline]`

Reply:
@christianpoveda you saw this comment? (just wondering as you reacted to the other one and then stopped, no rush)

Reply:
It seems I missed it :P